scripts/removejujuservices.bash

```diff
@@ -17,3 +17,6 @@ rm -rf /var/lib/juju/db/*
 
 echo "removing /var/lib/juju/raft/*"
 rm -rf /var/lib/juju/raft/*
+
+echo "removing /var/run/juju/*"
```

Comment:
We need to add this to _ cloudconfig/userdatacfg_unix.go_ as well.

Reply:
Fixed